js-envs-test-kit/src/test/scala/org/scalajs/jsenv/test/kit/TestEnv.scala

```diff
@@ -63,13 +63,13 @@ private[kit] class TestEnv private (
 
   val name: String = "TestEnv"
 
-  def start(input: Input, config: RunConfig): JSRun = {
+  def start(input: Seq[Input], config: RunConfig): JSRun = {
     require(msgs.isEmpty)
     callOnOutputStream(config)
     new TestRun
   }
 
-  def startWithCom(input: Input, config: RunConfig, onMessage: String => Unit): JSComRun = {
+  def startWithCom(input: Seq[Input], config: RunConfig, onMessage: String => Unit): JSComRun = {
```

Comment:
The two changes here are necessary because the file was not compiling as is. It turns out we were not running `jsEnvsTestKit/test` in the core repo!